src/watchmaker/__init__.py

```diff
@@ -108,30 +135,35 @@ def _get_config_data(self):
                     shutil.copyfileobj(response, outfile)
                 self.config_path = 'config.yaml'
             except urllib.error.URLError:
-                print('The URL used to get the user config.yaml file did not '
-                      'work!\nPlease make sure your config is available.')
+                logging.critical(
+                    'The URL used to get the user config.yaml file did not '
+                    'work!\nPlease make sure your config is available.'
```

Comment:
I agree with you on this front.  If we want it to be similar to typical logs, then most if not all actually keep info on one line per log.